java/src/jmri/jmrit/vsdecoder/VSDecoderPreferences.java

```diff
@@ -82,10 +81,10 @@ public VSDecoderPreferences(String sfile) {
         // Try to load preferences from the file
         try {
             root = prefs.rootFromFile(file);
-        } catch (java.io.FileNotFoundException e2) {
+        } catch (JDOMException | IOException e2) {
```

Comment:
This JDOMException probably needs to be in the next catch block. It will only be thrown if the IOException would not be.

Reply:
Thanks, fixed.